operator/controllers/utils/controller_utils.go

```diff
@@ -51,6 +51,25 @@ func GetAnnotation(mlDep *machinelearningv1.SeldonDeployment, annotationKey stri
 	}
 }
 
+// Get an annotation array from the Seldon Deployment given by annotationKey or return the array with fallback.
+func GetAnnotations(mlDep *machinelearningv1.SeldonDeployment, annotationKey string, separator string, fallback string) []string {
```

Comment:
I would rather avoid duplication of logic between `GetAnnotations` and `GetAnnotation` and rather introduce a function "HostsFromAnnotation" that does the split.